### DISCUSSION TITLE
Two locations to translate

### DISCUSSION THREAD

**Discussion by luisenriquereyes** [sircharlo/meeting-media-manager#246] [Translations]
Greetings again for the Dominican Republic.  The latest version 22.4.0 is very nice. Thanks for the new features like multiple congregations support.  Two locations that would be nice to have translated based on program language I have indicated in these two screen shots:

![Translate to Spanish](https://user-images.githubusercontent.com/78366076/162578704-c2042229-892d-43c9-850e-726c7f98c77a.jpg)

**Comment by sircharlo** [comment]
I agree! Unfortunately, I don't speak Spanish :)
Would you be able to take a look at these two files?

- https://github.com/sircharlo/jw-meeting-media-fetcher/blob/master/locales/es.json

Compare with:

- https://github.com/sircharlo/jw-meeting-media-fetcher/blob/master/locales/en.json

Could you copy all lines that are in the English version but missing in the Spanish version, and translate them?

If you can, thank you!

**Comment by luisenriquereyes** [comment]
Done.  You can find it here:  https://github.com/sircharlo/jw-meeting-media-fetcher/compare/master...luisenriquereyes:patch-1

Thank you for the opportunity to help 👍😁

**Reply by sircharlo** [reply]
And thank you for doing it!

**Comment by luisenriquereyes** [comment]
Hello my brother.  The latest update has now the "Enable button to present media..." option translated to Spanish now.  But the title "Choose your congregation" is still in English when the display language is in Spanish.  In the es.json file I translated, I see the field  "chooseCongregation"  says "Seleccione su congregación" which should be the correct translation.

**Reply by sircharlo** [reply]
You're absolutely right. Thanks for pointing that out!
